libs/auth-api-lib/src/lib/personal-representative/services/personalRepresentative.service.ts

```diff
@@ -225,7 +258,9 @@ export class PersonalRepresentativeService {
       return await this.sequelize.transaction(async (t) => {
         const newPr = await this.personalRepresentativeModel.create(
           { ...personalRepresentative },
-          { transaction: t },
+          {
+            transaction: t,
```

Comment:
~ You can also name the `t` param as `transaction` then simply do

```suggestion
            transaction,
```